tests/cli/test_cli.py

```diff
@@ -0,0 +1,505 @@
+);
+
+SELECT
+    id,
+    item_id,
+    'a' as new_col,
+    ds,
+FROM
+    sqlmesh_example.seed_model
+WHERE
+    ds between @start_ds and @end_ds
+"""
+        )
+
+
+def update_full_model(temp_dir) -> None:
+    with open(temp_dir / "models" / "full_model.sql", "w") as f:
+        f.write(
+            """
+MODEL (
+  name sqlmesh_example.full_model,
+  kind FULL,
+  cron '@daily',
+  grain item_id,
+  audits (assert_positive_order_ids),
+);
+
+SELECT
+  item_id,
+  count(distinct id) AS num_orders,
+FROM
+    sqlmesh_example.incremental_model
+GROUP BY item_id
+"""
+        )
+
+
+def assert_duckdb_test(result) -> None:
+    assert "Successfully Ran 1 tests against duckdb" in result.output
+
+
+def assert_new_env(result, new_env="prod", from_env="prod") -> None:
+    assert f"New environment `{new_env}` will be created from `{from_env}`" in result.output
+
+
+def assert_model_versions_created(result) -> None:
+    assert "All model versions have been created successfully" in result.output
+
+
+def assert_model_batches_executed(result) -> None:
+    assert "All model batches have been executed successfully" in result.output
+
+
+def assert_target_env_updated(result) -> None:
+    assert "The target environment has been updated successfully" in result.output
+
+
+def assert_backfill_success(result) -> None:
+    assert_model_versions_created(result)
+    assert_model_batches_executed(result)
+    assert_target_env_updated(result)
+
+
+def assert_plan_success(result, new_env="prod", from_env="prod") -> None:
+    assert result.exit_code == 0
+    assert_duckdb_test(result)
+    assert_new_env(result, new_env, from_env)
+    assert_backfill_success(result)
+
+
+def assert_virtual_update(result) -> None:
+    assert "Virtual Update executed successfully" in result.output
+
+
+def test_version(runner):
+    result = runner.invoke(cli, ["--version"])
+    assert result.exit_code == 0
+    assert re.fullmatch("\d+\.\d+\.\d+(\..*)?\n?", result.output)
+
+
+def test_plan_no_config(runner, tmp_path):
+    # Error if no SQLMesh project config is found
+    result = runner.invoke(cli, ["--paths", tmp_path, "plan"])
+    assert result.exit_code == 1
+    assert "Error: SQLMesh project config could not be found" in result.output
+
+
+def test_plan(runner, tmp_path):
+    create_example_project(tmp_path)
+
+    # Example project models have start dates, so there are no date prompts
+    # for the `prod` environment. User input `y` is required to apply the plan.
+    result = runner.invoke(cli, ["--paths", tmp_path, "plan"], input="y\n")
+    assert_plan_success(result)
+
+
+def test_plan_skip_tests(runner, tmp_path):
+    create_example_project(tmp_path)
+
+    # Successful test run message should not appear if `--skip-tests` is passed and plan is applied
+    result = runner.invoke(cli, ["--paths", tmp_path, "plan", "--skip-tests"], input="y\n")
+    assert result.exit_code == 0
+    assert "Successfully Ran 1 tests against duckdb" not in result.output
```

Comment:
I think we should assert this does show up in the `test_plan` test. Right now we could never print that tests passed and we wouldn't have a test fail here. 

Btw I'm saying to put it in `test_plan` since I was originally thinking "Oh have a test that makes sure that we see tests when `--skip-tests` is not passed in" but that would be exactly what `test_plan` is.

Reply:
`assert_plan_success()` covers that - it includes a test that "Successfully Ran 1 tests against duckdb" is in the output

Reply:
Ah sorry missed that. Thanks!